inbm/telemetry-agent/telemetry/telemetry_handling.py

```diff
@@ -202,23 +200,25 @@ def on_command(topic, payload, qos):
         cmd_lock.release()
 
 
-def get_dynamic_telemetry(with_docker: bool, rm_active: bool = False) -> Dict:
+def get_dynamic_telemetry(is_docker_installed: bool, rm_active: bool = False) -> Dict:
     """Collect dynamic telemetry in a dictionary.
 
-    @param with_docker: true if docker is on the system; else false.
+    @param is_docker_installed: true if docker is on the system; else false.
+    @param rm_active: true if resource manager is active; otherwise, false.
     @return: dynamic telemetry data
     """
     logger.debug("Collecting dynamic telemetry.")
 
-    # several different types possible in dictionary
-    dynamic_telemetry: Dict[str, Optional[Any]] = {'systemCpuPercent': get_cpu_percent(),
-                                                   'containersCpuPercent': get_docker_stats(with_docker),
-                                                   'availableMemory': get_available_memory(),
-                                                   'percentDiskUsed': get_percent_disk_used(),
-                                                   'coreTempCelsius': get_core_temp_celsius(),
-                                                   'networkInformation': get_network_telemetry(),
-                                                   'friendlyINBMVersionCommit': get_friendly_inbm_version_commit(),
-                                                   'friendlyINBMVisionVersionCommit': get_friendly_inbm_vision_version_commit()}
+    # several types possible in dictionary
+    dynamic_telemetry: Dict[str, Optional[Any]] \
+        = {'systemCpuPercent': get_cpu_percent(),
+           'containersCpuPercent': get_docker_stats(is_docker_installed),
```

Comment:
We are using the same keywords "containersCpuPercent"? As we are only returning CpuPercent and no other metrics(like memory usage etc?) In which case should it be changed?

Reply:
@tsirlapu  That's a separate story that was on the list that we talked about this morning.  This has an impact on both clouds and validation, so we need to do it separately.